tests/bluetooth/host/buf/mocks/fff.c

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/net/buf.h>
+#include <zephyr/bluetooth/buf.h>
+#include <mocks/net_buf.h>
```

Comment:
do we actually need all these headers?

Reply:
see above comment; this whole patch isn't needed anymore.